dashboard/src/utils/dateFunctions.js

```diff
@@ -1,11 +1,29 @@
+/**
+ * Bump Todate ahead by specified number of days
+ * @function
+ * @param {Date} date - Date to be bumped
+ * @param {number} numberOfDays - Number of days the date is to be bumped
+ * @returns {Date} - New Bumped date
+ */
 export const bumpToDate = (date, numberOfDays) => {
   return date.setUTCDate(date.getUTCDate() + numberOfDays);
 };
 
+/**
+ * Takes a string date and convert it into UTC Date object
+ * @function
+ * @param {string} strDate - Date in string format
```

Comment:
You should say something here about the expected format of the string; for example, we're depending on it *not* having any timezone information, or the `+ "Z"` will result in an error.